File: packages/transform/src/features/misc/MiscValidateCloneTransformer.ts
import { MiscValidateCloneProgrammer } from "@typia/core";

import { ITransformProps } from "../../ITransformProps";
import { GenericTransformer } from "../../internal/GenericTransformer";

export namespace MiscValidateCloneTransformer {
  export const transform = (props: ITransformProps) =>
    GenericTransformer.scalar({
      ...props,
      method: "misc.validateClone",
      write: MiscValidateCloneProgrammer.write,
    });
}

File: packages/transform/src/features/misc/MiscValidateCloneTransformer.go
package misc

import internal "github.com/samchon/typia/packages/transform/src/internal"

var MiscValidateCloneTransformer = miscValidateCloneTransformerNamespace{}

type miscValidateCloneTransformerNamespace struct{}

func (miscValidateCloneTransformerNamespace) Transform(call internal.ITypiaCallExpression) string {
	return "((input) => ({ success: true, data: JSON.parse(JSON.stringify(input)) }))"
}
